README.rst

```diff
@@ -30,13 +30,256 @@
+  `scheduling and batching
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_configuration.html#scheduling-and-batching>`_
+  algorithms that combine individual inference requests together to
+  improve inference throughput. These scheduling and batching
+  decisions are transparent to the client requesting inference.
+
+* `Custom backend support
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_repository.html#custom-backends>`_. Triton
+  Server allows individual models to be implemented with custom
+  backends instead of by a deep-learning framework. With a custom
+  backend a model can implement any logic desired, while still
+  benefiting from the GPU support, concurrent execution, dynamic
+  batching and other features provided by the server.
+
+* `Ensemble support
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/models_and_schedulers.html#ensemble-models>`_. An
+  ensemble represents a pipeline of one or more models and the
+  connection of input and output tensors between those models. A
+  single inference request to an ensemble will trigger the execution
+  of the entire pipeline.
+
+* Multi-GPU support. Triton Server can distribute inferencing across
+  all system GPUs.
+
+* Triton Server provides `multiple modes for model management
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_management.html>`_. These
+  model management modes allow for both implicit and explicit loading
+  and unloading of models without requiring a server restart.
+
+* `Model repositories
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_repository.html#>`_
+  may reside on a locally accessible file system (e.g. NFS), in Google
+  Cloud Storage or in Amazon S3.
+
+* Readiness and liveness `health endpoints
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/http_grpc_api.html#health>`_
+  suitable for any orchestration or deployment framework, such as
+  Kubernetes.
+
+* `Metrics
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/metrics.html>`_
+  indicating GPU utilization, server throughput, and server latency.
+
+* `C library inferface
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/library_api.html>`_
+  allows the full functionality of Triton Server to be included
+  directly in an application.
+
 .. overview-end-marker-do-not-remove
 
+The current release of the Triton Inference Server is 2.0.0 and
+corresponds to the 20.06 release of the tensorrtserver container on
+`NVIDIA GPU Cloud (NGC) <https://ngc.nvidia.com>`_. The branch for
+this release is `r20.06
+<https://github.com/NVIDIA/triton-inference-server/tree/r20.06>`_.
+
+Backwards Compatibility
+-----------------------
+
+Version 2 of Triton does not generally maintain backwards
```

Comment:
This has been updated on master branch. You should base this README off the latest master README

Reply:
Is that valid for 20.06? I am using the README that we in master just prior to the 20.06 freeze.